lib/consoletest.pm

```diff
@@ -23,18 +23,9 @@ sub post_fail_hook {
     select_console('log-console');
     $self->SUPER::post_fail_hook;
     $self->remount_tmp_if_ro;
-    # Export logs after failure
-    assert_script_run("journalctl --no-pager -b 0 > /tmp/full_journal.log");
-    upload_journal "/tmp/full_journal.log";
-    assert_script_run("dmesg > /tmp/dmesg.log");
-    upload_logs "/tmp/dmesg.log";
+    $self->export_basic_logs;
```

Comment:
I would change this one and the others to export_logs_$type to keep it in sync with the export_logs_locale method introduced few days ago :)